bazel/repository.bzl

```diff
@@ -0,0 +1,80 @@
+load("@bazel_tools//tools/build_defs/repo:utils.bzl", "maybe")
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+def opentelemetry_cpp_deps():
+    """Loads dependencies need to compile the opentelemetry-cpp library."""
+
+    # Google Benchmark library.
+    # Only needed for benchmarks, not to build the OpenTelemetry library.
+    maybe(
+        native.local_repository,
```

Comment:
Are consumers of otel-cpp meant to load e.g. benchmark before calling `opentelemetry_cpp_deps()`?

Reply:
yeah.  I need to write the "INSTALLING" section for bazel, but basic gist is if you want to configure your own version/build that you can do so if you load BEFORE calling our macro, but you'll be on your own ensuring the version is compatible with us.